data/brands/shop/supermarket.json

```
@@ -6736,6 +6736,25 @@
         "shop": "supermarket"
       }
     },
+    {
+      "displayName": "U購Select U Select",
+      "locationSet": {"include": ["hk"]},
+      "tags": {
+        "brand": "U購Select U Select",
+        "brand:en": "ParknShop",
```

```suggestion
        "brand:en": "U Select",
```

Maybe mixed with “百佳超級市場 PARKnSHOP” ？